api/types/license.go

```diff
@@ -28,11 +28,16 @@ import (
 type License interface {
 	Resource
 
-	// GetReportsUsage returns true if teleport cluster reports usage
-	// to control plane
+	// GetReportsUsage returns true if the Teleport cluster should report usage
+	// to the Houston control plane.
 	GetReportsUsage() Bool
-	// SetReportsUsage sets usage report
+	// SetReportsUsage sets the Houston usage reporting flag.
 	SetReportsUsage(Bool)
+	// GetUsageReporting returns true if the Teleport cluster should report
+	// usage to Sales Center.
+	GetUsageReporting() Bool
+	// SetUsageReporting sets the Sales Center usage reporting flag.
+	SetUsageReporting(Bool)
```

Comment:
+1 for more explicit names, maybe mention Houston and SalesCenter in the name so it's extra clear?

Reply:
076be5c

Reply:
Could we mention Houston explicitly as well?

Reply:
> Could we mention Houston explicitly as well?
I don't want to spend the time for that now, as it will require some changes both in `teleport.e` and in `cloud` and the old `usage` flag is one that we might actually be able to outright remove entirely as part of gravitational/teleport.e#892 (since nothing at that point should be producing licenses with that flag or discriminating on licenses based on that flag).

Reply:
Fair enough.